core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -445,6 +445,29 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("SPARK-33084: add jar when path contains comma") {
```

Comment:
Do we really need to support this `comma`?

Reply:
> Do we really need to support this `comma`?

This is a UT added earlier.
In current code seems we don't need it?

Reply:
Thanks. Yes. Please remove this.

Reply:
Done